docs/getting-started/prerequisites.md

```diff
@@ -7,67 +7,67 @@ permalink: /getting-started/prerequisites/
 ---
 
 # Prerequisites
+{: .no_toc}
+
+This page explains how to set up your environment for using ClusterFuzz.
 
 - TOC
 {:toc}
 
 ---
 ## Requirements
 Many features of ClusterFuzz depend on [Google Cloud
-Platform](https://cloud.google.com) services (see
-[this]({{ site.baseurl }}/architecture/#requirements) page for more details).
-However, it's possible to run it locally without these dependencies for testing
-purposes.
+Platform](https://cloud.google.com) services, but it's possible to run it locally without these dependencies for testing purposes. See the [Architecture page]({{ site.baseurl }}/architecture/#requirements) for more details.
 
-While ClusterFuzz runs on a number of platforms, local development is only
-supported on **Linux** and **macOS**.
+**Note:** Local development is only supported on **Linux** and **macOS**.
 
 ## Getting the code
+
+Clone the ClusterFuzz repository to your machine by running the following command:
```

Comment:
meh, I suspect that bash commands are understandable on their own to our target audience

Reply:
I'm sure you're right - just for completeness :)